src/components/shared/Layout/PageContent.tsx

```diff
@@ -12,18 +12,21 @@ export interface PageContentProps extends PropsWithChildren {
 export function PageContent(props: PageContentProps) {
   const { children, isPaper, viewHeight, hiddenHeader, maxWidth, sx } = props;
 
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
+
   return (
     <Container
       component={isPaper ? Paper : "div"}
-      maxWidth={maxWidth ?? "xl"}
+      maxWidth={maxWidth ?? false}
```

Comment:
I would also revert the max width changes - it makes things stretch on larger screens
![WithoutMaxWidth](https://github.com/user-attachments/assets/2d7a8e11-05c7-4cf1-96bb-f25b1a2f7492)
![WithMaxWidth](https://github.com/user-attachments/assets/e9b44947-1585-410b-b92a-683ed682eb1c)

Reply:
reverted